### PR TITLE
Brian branch

### PR DESCRIPTION
**Pull request by Brianified**

### CODE REVIEW

File: src/server/server.py [hunk 1]
```diff
@@ -1,333 +1,48 @@
...
+    options.parse_command_line()
 
-    def stop(self):
-        """
-        Stop server
-        """
-        self.sock.close()
+    # create logger for app
+    logger = logging.getLogger('app')
+    logger.setLevel(logging.INFO)
```
Comment (SeraphShroud) [inline src/server/server.py]: There is a logging_config already in place under logging_config.ini so you don't need to set this up here. You can just call logger = logging.getLogger().

File: src/server/service/draft_service.py [hunk 2]
```diff
@@ -0,0 +1,38 @@
+from src.server.ygo_card_db_service import YGOCardDBService
+
+class InvalidCreationError(Exception):
```
Comment (SeraphShroud) [inline src/server/service/draft_service.py:3]: Can you make an exceptions.py file and add these exceptions? I'd like to keep all the exceptions in a single file for ease of modifications.

**Review by SeraphShroud — CHANGES_REQUESTED** [review]
Can you edit the readme to remove mention of the server-client demo and put how to run the server with the websocket? Specifically the steps on how to start up the server and connect to the rooms (basically what you showed me in the demo)

Please install flake8 linter. You can autoformat documents by right clicking the code and clicking "Format Document" once you have it installed. It will probably require you to pip3 install autopep8.

I saw there is an app/ directory and a service/ directory. Do we need the app/ directory still or can we just move the files up? Can you consolidate the current services in the service directory?

Do we still need all the tic-tac-toe files?

**Comment by Brianified** [comment]
> Can you edit the readme to remove mention of the server-client demo and put how to run the server with the websocket? Specifically the steps on how to start up the server and connect to the rooms (basically what you showed me in the demo)
> 
> Please install flake8 linter. You can autoformat documents by right clicking the code and clicking "Format Document" once you have it installed. It will probably require you to pip3 install autopep8.
> 
> I saw there is an app/ directory and a service/ directory. Do we need the app/ directory still or can we just move the files up? Can you consolidate the current services in the service directory?
> 
> Do we still need all the tic-tac-toe files?

app essentially handles the communication and service is the are the python scripts that do all the work for the game. I wanted to separate that distinction

**Review by SeraphShroud — CHANGES_REQUESTED** [review]
Once the imports are resolved and we can test the server I'll approve

**Review by SeraphShroud — APPROVED** [review]

**Pull request merged**